dummy_core/index.js

```diff
@@ -0,0 +1,46 @@
+const WebSocket = require('ws');
+
+const wss = new WebSocket.Server({ port: 3000 });
+
+wss.on('connection', function connection(ws) {
+
+    console.log("Got connection!");
+
+    ws.on('message', async function incoming(data) {
+        try {
+            let obj = JSON.parse(data);
+            // Uncomment to test messages FROM client (may do it hard to enter commands)
+            // console.log(JSON.stringify(obj));
+        } catch(e){
+            console.log("Received invalid data: %s", data);
```

Comment:
Here's a nice way to format strings:
```javascript
`Received invalid data: ${data}`
```
I saw later that you're using it for html below, but it works for normal strings too.